Cargo.toml

```diff
@@ -22,8 +22,7 @@ bytes = "1"
 argh = "0.1"
 crossterm = "0.27"
 flate2 = "1.0"
-futures = "0.3.29"
-futures-core = "0.3"
```

Comment:
You should not get rid of `futures-core`, as that is where you'll find stuff like the `TryStream` trait. And as you'll see it's also still a dependency of futures-lite: https://crates.io/crates/futures-lite/2.3.0/dependencies